dev-server/mock-routes.js

```diff
@@ -51,4 +51,21 @@ app.get('/familie-ba-sak/api/logg/2', (req, res) => {
     setTimeout(() => res.send(lesMockFil(`logg-2.json`)), delayMs);
 });
 
+app.get('/familie-ba-sak/api/fagsaker/sok', (req, res) => {
+    if (req.query.personIdent === '17058018783') {
+        setTimeout(() => res.send(lesMockFil(`søk-2.json`)), 500);
+    } else if (req.query.personIdent === '28111883612') {
+        setTimeout(() => res.send(lesMockFil(`søk-1.json`)), 500);
+    } else {
```

Comment:
Are these real pin's? They look a bit to real. Consider using dummy data (12345678910).

Reply:
Generated with Dolly. Is it ok to use?

Reply:
Ok, I think so 👍